esi_bot/processor.py

```diff
@@ -40,8 +40,8 @@
 )
 
 REACTION_TRIGGERS = {
-    re.compile(r"(^|.*( |!))crest( |$|\!|\?|\.)", re.IGNORECASE): "rip",
-    re.compile(r"(^|.*( |!))xml(api)?( |$|\!|\?|\.)", re.IGNORECASE): "wreck",
+    # re.compile(r"(^|.*( |!))crest( |$|\!|\?|\.)", re.IGNORECASE): "rip",
```

Comment:
this was voted down

https://www.strawpoll.me/15876652/r

lets leave it in for now, not really doing any harm as is

Reply:
@a-tal undone.